cmake/detect-features-c.cmake

```diff
@@ -11,6 +11,9 @@ else()
 endif()
 set(CMAKE_REQUIRED_DEFINITIONS -D_GNU_SOURCE)
 
+string(REPLACE " " ";" CMAKE_REQUIRED_INCLUDES_add "${ADDITIONAL_INCLUDE_DIRS}")
+set(CMAKE_REQUIRED_INCLUDES ${CMAKE_REQUIRED_INCLUDES_add} ${CMAKE_REQUIRED_INCLUDES_add})
```

Comment:
Why is `CMAKE_REQUIRED_INCLUDES_add` duplicated here?

Reply:
good catch, fixed.